### PR TITLE
Add lights off option

### PR DESCRIPTION
**Pull request by countnewb**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2375)

### CODE REVIEW

File: english/lightsoff/lightsoff.md [hunk 1]
```diff
@@ -0,0 +1 @@
+It is pitch black. You are likely to be eaten by a grue.
```
Comment (casey-collab) [inline english/lightsoff/lightsoff.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by countnewb** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :checkered_flag:

**Pull request merged**